client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -1,144 +1,185 @@
-import { useEffect, useState } from 'react';
+import { useEffect } from 'react';
 import create from 'zustand';
 import { AppRoute } from '@openmsupply-client/config';
 import {
   AuthenticationError,
-  AuthError,
   LocaleKey,
-  LocalStorage,
-  ServerStatus,
+  SyncStateType,
+  TypedTFunction,
   useNavigate,
   useTranslation,
 } from '@openmsupply-client/common';
 import { useHost } from '../../api/hooks';
 
-const SERVER_RESTART_TIMEOUT = 60000;
-const POLLING_INTERVAL = 3000;
-const POLLING_DELAY = 6000;
+// For refetch of syncStatus and syncState
+const POLLING_INTERVAL = 500;
```

Comment:
I always wonder when there's a comment above a variable ( or CONST ) whether the var name could just be better and the comment removed 😉 

`STATUS_POLLING_INTERVAL`?

and below, `DEFAULT_SYNC_INTERVAL`?

Reply:
Great idea 🙏 4849e03ebc5e12d36af97f050e1dad2aae1b3390